### DISCUSSION TITLE
Déploiement du fichier default.v0.transmission.yml

### DISCUSSION THREAD

**Discussion by MJMESS** [MTESSDev/FRW#35] [Annonces]
Bonjour, 

pour déployer un fichier qui est à la racine, est-ce que la façon de faire est d'ajouter une tâche de copie file pour ce fichier dans chacun des pipelines de déploiements des différents formulaires ?

Merci à l'avance!

**Comment by anisite** [comment]
Bonsoir, il n'est pas recommandé de déployer les fichiers à la racine dans tous les pipelines de tous les formulaires. Idéalement, une équipe qui a monté un pipeline pour chaque formulaire devrait s'ajouter un pipeline "base" ou "config" peu importe le nom pour monter uniquement les fichiers de la racine (niveau système) indépendamment du reste. Il n'y a pas juste une bonne façon de faire, mais c'est celle qui nous semble la meilleure. 

Revenez-moi s'il y a des enjeux ou autres, merci

**Comment by MJMESS** [comment]
Parfait merci!
